src/content/1.2/code/ocaml/snippet05.ml

```diff
@@ -1 +1,14 @@
-let fact n = List.fold (List.range 1 n) ~init:1 ~f:( * )
+(* OCaml doesn't ship with a lazy sequence range syntax nor a product
```

Comment:
👍 I think I used Janestreet libs in a few places to reduce the snippet size and to move ahead quickly, but thanks for removing the dependency!